lib/stimulus_reflex/broadcasters/selector_broadcaster.rb

```diff
@@ -64,5 +58,26 @@ def selector?
     def to_s
       "Selector"
     end
+
+    private
+
+    def create_update_collection(selectors, html)
+      updates = selectors.is_a?(Hash) ? selectors : {selectors => html}
+      updates.map do |key, value|
+        Update.new(key, value, reflex)
+      end
+    end
+
+    StimulusReflex::SelectorBroadcaster::Update = Struct.new(:key, :value, :reflex) do
```

Comment:
Why not just inline the `Struct.new` on line 67?

Not saying you should, just learning from your approach.

Reply:
could probably do so, yes... I thought it'd better have a name attached to it?

Reply:
I find the syntax really interesting. It reminds me of function passing in JavaScript.

I suppose the main reason to do this is code readability and ease of future changes. I'm just glad this could be done within the same file. I find it generally much harder to follow when things get split across multiple files.